packages/p2p/src/components/orders/order-table/order-table-content.jsx

```diff
@@ -48,7 +44,7 @@ const OrderTableContent = ({ showDetails, is_active }) => {
                 limit: list_item_limit,
                 active: is_active_tab ? 1 : 0,
             }).then(response => {
-                if (is_mounted) {
+                if (isMounted.current) {
```

Comment:
There's a `setIsLoading` on line 57 which should be inside this `is_mounted` check.